common/messaging/kafkaConfig.go

```diff
@@ -30,6 +30,7 @@ type (
 	// KafkaConfig describes the configuration needed to connect to all kafka clusters
 	KafkaConfig struct {
 		TLS      auth.TLS                 `yaml:"tls"`
```

Comment:
I feel like this should be per cluster instead of global. But not very familar with the actual usage.